Fix:

error: use of undeclared identifier 'calloc'
error: use of undeclared identifier 'malloc'
--- fontforge/woff2.cc.orig	2022-12-31 23:25:39.000000000 -0600
+++ fontforge/woff2.cc	2023-08-10 20:32:49.000000000 -0500
@@ -22,6 +22,7 @@
  * ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <stdlib.h>
 #include <woff2/decode.h>
 #include <woff2/encode.h>
 
